README.md

```diff
@@ -252,6 +252,32 @@ make container
 Simply build and run kube-state-metrics inside a Kubernetes pod which has a
 service account token that has read-only access to the Kubernetes cluster.
 
+### Prometheus Operator
+
+The Prometheus Operator ([`kube-prometheus`](https://github.com/prometheus-operator/kube-prometheus/)) installs kube-state-metrics as one of its [components](https://github.com/prometheus-operator/kube-prometheus#kube-prometheus); you do not need to install kube-state-metrics if you're using the Prometheus Operator.
+
+The kube-state-metrics Deployment [manifest] overrides several command line arguments of the kube-state-metrics binary:
+
+```YAML
+containers:
+- name: kube-state-metrics
+  args:
+  - --host=127.0.0.1             # Default: "::"
+  - --port=8081                  # Default: "8080"
+  - --telemetry-host=127.0.0.1   # Default: "::"
+  - --telemetry-port=8082        # Default: "8081"
+  image: k8s.gcr.io/kube-state-metrics/kube-state-metrics:v1.9.8
```

Comment:
Master should not reference 1.9.8 images.

Reply:
If I change this to say `image: k8s.gcr.io/kube-state-metrics/kube-state-metrics` would that be OK?

Reply:
Not sure this step is necessary as people have a tendency of copy pasteing yaml around so I would prefer if they did it from working examples. If that makes sense? :) 

Also we autogenerate all our code, so adding things like this would mean manual updating, we can embed the code instead if you think its important to reference it here?

Reply:
kube-prometheus runs kube-state-metrics but it does so without specifying `--resources`.

As a result, only the default resources are enabled.

If the users wishes to enable non-default metrics (e.g. VPA), they must include the `--resources` flag, list all the default resources and include the non-default metric(s).

Perhaps replace the text with some variant of that explanation?

Reply:
Yes, that is out of the box, but you can add a jsonnet patch for this if you are using kube-prometheus we have that documented in there -> https://github.com/prometheus-operator/kube-prometheus#customizing-kube-prometheus

I would suggest to revise this to "Out of the box kube-promethus ships with default mixin configuration if you want to customize have a look at <https://github.com/prometheus-operator/kube-prometheus#customizing-kube-prometheus>" What do you think?

Reply:
Fixed: https://github.com/kubernetes/kube-state-metrics/pull/1412/commits/86689d45db2730aff799e495729f29993734087c